docs/metasploit-framework.wiki/Attacking-ADCS-ESC-Vulnerabilities.md

```diff
@@ -0,0 +1,916 @@
+# Workflow
+```mermaid
```

Comment:
Sounds good, I can add in some definitions so long to this PR since I wrote up some explanations for myself in my Obsidian notes. Can probably use those to craft some definitions for these three attacks plus can provide a few others. Can't say I've gone through all of them yet though, but all of the ones we presently have support for I have notes on.

If you'd rather that I relocate these elsewhere though please feel free to suggest a location; can use the one above or some other file if that is more suitable.

Reply:
Added in an improved explanation at 93f2e472bcdd53127e00e3ed0ec0fd4711799334, let me know if that works for you.